src/services/membership.js

```diff
@@ -53,7 +53,7 @@ import http from './http';
  * @return {Promise<any>} Response
  */
 function addMembership(data) {
-  return http.post(`memberships`, data).catch(reason => {
+  return http.post('memberships', data).catch(reason => {
     console.log(reason);
```

Comment:
was this just a "bunny-trail" change, irrelevant to the error log?

Reply:
Yes, this one was just a change for consistency, as the special back tick characters only need to be used when there is a multi-line string or a variable/expression inside the string.
It's not relevant to error logging.